docs/fieldnorm/life_cycle.md

```diff
@@ -1,7 +1,56 @@
+One of the main responsibilities of Bioinformaticians within a diagnostic setting involves the development and maintenance of software used in diagnostic processes. Because ISO-15189 does not cover software development, it is essential that inhouse software is created and maintained in accordance with recognized industry standards, such as IEC-62304 and IVDR appendix I. The IEC-62304 description of the software development life cycle (SDLC) is consistent with the practices in most software-centric entities. SDLC processes that are commonly used include planning, requirements, architectural design, development, testing, deployment and maintenance. Bioinformatics software development within a ISO-15189 certified genetics laboratory should establish clear guidelines that cover all SDLC stages. 
 
-Donec mi mi, suscipit venenatis rutrum eu, gravida consectetur justo. Nunc neque ipsum, tincidunt ac libero eu, tincidunt accumsan lorem. Ut aliquet maximus nibh, id volutpat nisi efficitur nec. Nulla et feugiat ipsum. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Curabitur egestas est ac augue hendrerit finibus. Pellentesque hendrerit, magna gravida tincidunt bibendum, velit leo finibus nisi, non sagittis dui purus vitae orci. Etiam lobortis diam sed sem congue lacinia. Phasellus in fermentum libero.
+ Planning
+Planning entails developing a comprehensive software development plan. This includes brainstorming the initial ideas, ideation of the final goal, determining the necessary resources, creating a schedule and establishing achievable milestones. 
+Within a genetics laboratory software development often occurs within the background of laboratory tests. The laboratory therefore needs to develop a solid strategy for change management for its genetic tests that emphasizes all these features, and software development should be an integral part of this strategy. 
+Smaller scale software development and maintenance cycles that might not affect other areas of a genetic lab should maintain a similar strategy on a smaller scale and less laborious, and might use tooling that is more effective for software development, such as code based documentation, an online documentation system that are more suitable for software development planning (like confluence, or wikipages), an issue management system (like issue manager or jira) or combinations thereof.
```

Comment:
Not a comment to change but just to approve the need to have "big scale" and "small scale" approache when it comes to life cycle of software. I cannot imagine putting the same burden on a small maintenance script than a big multi part LIMS when it comes to track changes. Should we be able to describe what make something big or small scale ?

Reply:
yes; maybe in terms of "new major feature" or "new software" or adding changes. 
I would not want a strong link with the wetlab; as in... we won't get them into 'our way of working'. 
Connection with the lab should be in requirements, (clinical) validation, risk analysis, communication

Reply:
Maybe we should mention that we need collaboration in terms of "intended use" "requirements" etc from the laboratory. That the laboratory should be in the SDLC in terms of checking whether whatever is build works 
It should be a collaboration, but they need to let us do the software development /bioinformatics.

Reply:
I need a definition of small and large scale

--> note: maybe we need a list of abreviation and defintions ;)

Reply:
I have added that large and small scale projects should be defined in guidelines with a note giving examples of what large scale and small scale could be.